app/models/company.rb

```diff
@@ -6,6 +6,9 @@ class Company < ActiveRecord::Base
   validates :title,           presence: true
   validates :description,     presence: true
   validates :about,           presence: true
-  validates :address,         presence: true
+  validates :address1,        presence: true
+  validates :city,            presence: true
+  validates :state,           presence: true
```

Comment:
@godfreym american states are 2 capital letters. you should be able to find a list so you can make a dropdown.